var/spack/repos/builtin/packages/libmesh/package.py

```diff
@@ -35,8 +35,22 @@ class Libmesh(Package):
 
     version('1.0.0', 'cb464fc63ea0b71b1e69fa3f5d4f93a4')
 
+    variant('mpi', default=True, description='Enables MPI parallelism')
+
+    depends_on('mpi', when='+mpi')
+
+    # Parallel version of libmesh needs parallel solvers
+    depends_on('petsc+mpi', when='+mpi')
```

Comment:
I have not tried the serial version but I think it does have builtin solvers. In any case, my changes should not have affected its behavior.